pkg/sdn/api/v1beta1/types.go

```diff
@@ -0,0 +1,31 @@
+package v1beta1
+
+import (
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1beta3"
+)
+
+type ClusterNetwork struct {
+	kapi.TypeMeta   `json:",inline"`
+	kapi.ObjectMeta `json:"metadata,omitempty"`
+
+	Network          string `json:",network"`
```

Comment:
Externally serialized fields in the API need to have a description in order to support generation of the swagger API docs.

For an example, see https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta1/types.go#L61

Please include a description on each field you serialize in the API